xapian-applications/omega/query.cc

```diff
@@ -1940,6 +1955,19 @@ eval(const string &fmt, const vector<string> &param)
 		value = args[0];
 		url_prettify(value);
 		break;
+	    case CMD_qid: {
+		string qstr = args[0];
+		trim_string(qstr);
```

Comment:
This is to save us from assigning different IDs to some queries that are essentially the same (queries with accidental leading or trailing spaces e.g. "string   " and "  string"). Also, as it isn't useful to log such queries with different IDs, I think it would help minimise noise in the log data.